src/cript/api/api.py

```diff
@@ -603,7 +604,7 @@ def _get_db_schema(self) -> dict:
     @beartype
     def _is_node_schema_valid(self, node_json: str, is_patch: bool = False) -> bool:
```

Comment:
I would write a `force_schema_validation` argument into this function.
This way, there is a way to force the validation.

And the we can force this validation in `api._internal_save`.

That looks like the best solution to me.

Reply:
added